blockbase/experimental-theme.json

```diff
@@ -0,0 +1,28 @@
+{
+	"global": {
+		"settings": {
+			"color": {
+				"gradients": [ ],
+				"palette": [ ]
+			},
+			"typography": {
+				"customFontSize": true,
+				"customLineHeight": true,
+				"fontSizes": [ ],
+				"spacing": {
+					"customPadding": true,
+					"units": [ "px", "em", "rem", "vh", "vw" ]
+				}
+			},
+			"custom": {
```

Comment:
These rules are only used in the alignments stylesheets above, so if those are removed, these could be removed too.